public/webpack.config.js

```diff
@@ -14,17 +15,21 @@ module.exports = {
 	resolve: {
 		modulesDirectories: ['node_modules', 'bower_components'],
 		alias: {
-			'specific': __dirname + '/app/specific'
+			'setup': __dirname + '/app/project-setup',
+			'actions': __dirname + '/app/actions/actions',
+			'screens': __dirname + '/app/screens',
+			'stores': __dirname + '/app/stores',
+			'common': __dirname + '/app/common',
+			'mixins': __dirname + '/app/common/mixins'
 		},
-        extensions: ['', '.js', '.jsx']
+    extensions: ['', '.js', '.jsx']
 	},
 	module: {
 		loaders: [
-            {test: /\.jsx$/, loader: 'jsx-loader?insertPragma=React.DOM&harmony'},
-			{test: /\.scss$/, loader: 'css!autoprefixer!sass?' +
-				'includePaths[]=' + __dirname + '/app/specific' +
-				'&includePaths[]=' + __dirname + '/bower_components/foundation/scss'},
-
+			{test: /\.jsx$/, loader: 'jsx-loader?insertPragma=React.DOM&harmony'},
+			{test: /\.scss$/, loader: ExtractTextPlugin.extract('style','css!autoprefixer!sass?' +
```

Comment:
It's experiment, we can use just a simple loader and attache styles to the DOM (what it not so good for prod case)